addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2657,3 +2657,27 @@ export function cleanZWS(node) {
         .filter(node => node.nodeType === Node.TEXT_NODE)
         .forEach(node => node.nodeValue = node.nodeValue.replace(/\u200B/g, ''));
 }
+
+/**
+ * check click or mouse down event occurred inside the checkbox box.
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+export function isMouseInsideCheckbox(ev) {
+    const node = ev.target;
+    const beforStyle = window.getComputedStyle(node, ':before');
```

Comment:
spell check `beforeStyle`

Reply:
I agree. Even if it's a copy-paste, you should take the opportunity to correct the typo.